boards/nxp/mcxw72_evk/doc/index.rst

```diff
@@ -0,0 +1,146 @@
+.. zephyr:board:: mcxw72_evk
+
+Overview
+********
+
+The MCXW72-EVK
+
+The MCX W72x family features a 96 MHz Arm® Cortex®-M33 core coupled with a
+multiprotocol radio subsystem supporting Matter, Thread, Zigbee and
+Bluetooth LE. The independent radio subsystem, with a dedicated core and
+memory, offloads the main CPU, preserving it for the primary application and
+allowing firmware updates to support future wireless standards.
+
+Hardware
+********
+
+- MCXW72 Arm Cortex-M33 microcontroller running up to 96 MHz
+- 2MB on-chip Flash memory unit
+- 256 KB TCM RAM
+- On-board MCU-Link debugger with CMSIS-DAP
+
+For more information about the MCXW72 SoC and MCXW72-EVK board, see:
+
+- `MCXW72 SoC Website`_
+- `MCXW72-EVK Website`_
+
+Supported Features
+==================
+
+The ``mcxw72_evk`` board target in Zephyr currently supports the following features:
```

Comment:
needs full board target i.e. with cpu0 or cpu1 or drop target to mean board as a whole

Reply:
I have not addressed this comment on the most recent push, I still plan to address this comment.

Reply:
CPU1 Support will be added in the near future.